### ISSUE TITLE
Configuration option that closes the panel when debugging ends

### ISSUE THREAD

**Issue by Fell** [microsoft/vscode#100182]
In https://github.com/microsoft/vscode/issues/99050 a bug was fixed, changing the behaviour of `debug.console.closeOnEnd`. Before 1.46, the whole panel containing the debug console was closed. Now, only the debug console closes, which is expected.

However, I found it quite convienient that the whole panel was closed after debugging. I keep things like the watch window and the call stack in there, which require a lot of horizontal space.

Any chance of making that an option?

tl;dr:  I want `debug.console.closeOnEnd`, but for the panel.

**Comment by isidorn** [comment]
This is a fair feature request, however we do not have it on our immediate plan. Thus assigning to backlog and we might add this in the future. Thanks
